aiohue/v1/api.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import annotations
 
+# ruff: noqa: D100,D101,D102,D105,D107,PLR0913
```

Comment:
I suggest excluding these rules in the general ruff for now.

Reply:
Well, I do want to make a general exception.
The v1 code is legacy code we do not touch anymore (hence these exceptions). The v2 code should be properly linted and up to date with code standards

Reply:
See my suggestion on `pyproject.toml` below